site/mkdocs.yml

```diff
@@ -75,7 +75,9 @@ nav:
       - Maintenance Procedures: spark-procedures.md
       - Structured Streaming: spark-structured-streaming.md
       - Time Travel: spark-queries/#time-travel
-    - Flink: flink.md
+    - Flink:
+      - Getting Started: flink.md
```

Comment:
I am thinking we might want to break down the flink documentation like Spark (no need to do in this PR), just to make it more consistent between the 2 and easier to navigate. What do you think?

Reply:
Yes,  I think it's necessary to break the flink into more pages so that people could get the correct point in the separate page. We will also introduce more feature for iceberg flink integration work, such as flink table maintaince API ,  flip-27 unified source/sink etc. 

I will prefer to publish a separate PR to address the break-down thing, let this one focus on adding the iceberg flink connector.